core/src/include/jcr.h

```diff
@@ -160,18 +160,15 @@ class JobControlRecord {
   uint32_t JobFiles{};          /**< Number of files written, this job */
   uint32_t JobErrors{};         /**< Number of non-fatal errors this job */
   uint32_t JobWarnings{};       /**< Number of warning messages */
-  uint32_t AverageRate{};       /**< Last average bytes/sec */
-  uint32_t LastRate{};            /**< Last sample bytes/sec */
+  uint32_t AverageRate{};       /**< Average bytes/sec */
```

Comment:
You can also remove `AverageRate` and `UpdateJobStats` now.  Both of them are only used once and `UpdateJobStats` barely does anything now.   Just compute the value when its supposed to get printed.